lib/analyzer/package-managers/apt.ts

```diff
@@ -22,6 +22,24 @@ export function analyze(
   });
 }
 
+export function analyzeDistroless(
+  targetImage: string,
+  aptFiles: string[],
```

Comment:
should the name actually be `dpkgFiles`? in general in this pr wherever it says apt we might should say dpkg?

Reply:
if we want to make a more broad change, then yes.
otherwise I think we should be consistent with the current codebase.
happy to consider a minor refactor/reword effort after we get rid of the dynamic scanning code